tests/inputs/test_inputs.py

```diff
@@ -80,6 +81,57 @@ def test_remote_indir(tmp_path, valid_http_url):
     assert fpath.parent == tmp_path
 
 
+def test_remote_default_user_agent(valid_http_url, monkeypatch):
+    def mock_stream_file(**kwargs):
+        headers = kwargs.get("headers")
+        assert headers is not None
+        assert isinstance(headers.get("User-Agent"), str)
+
+    monkeypatch.setattr(
+        zimscraperlib.inputs,  # pyright: ignore
```

Comment:
Please try to always be more specific in the rules that have to be ignored.
Here `# pyright: ignore[reportAttributeAccessIssue]` is sufficient, it helps nail other issues which might appears + help in understanding why the hint has been used. I agree we need it.